contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -4559,8 +4617,30 @@ load_categories_hash(RawStmt *cats_sql, const char * sourcetext, MemoryContext p
 		/* internal error */
 		elog(ERROR, "load_categories_hash: SPI_connect returned %d", ret);
 
-	/* Retrieve the category name rows */
-	ret = SPI_execute_raw_parsetree(cats_sql, sourcetext, true, 0);
+	/* 
+	 * Retrieve the category name rows    
+	 *
+	 * If view_uuid is defined, then we will use the provided uuid to form the catefory_sql and 
+	 * get result. category_sql name format is pvt_cv_[uuid]
+	 * 
+	 * if view_uuid is not defined, then we will use the rawparsetree of catefory_sql to get the
+	 * result
+	 */
+	if (!view_uuid || strlen(view_uuid) != 0)
```

Comment:
same here. `if (!view_uuid || strlen(view_uuid) != 0)` seems wrong to me

Reply:
my dumb mistake😥